rebar.config.erl

```diff
@@ -194,7 +194,7 @@ overlay_vars_pkg(bin) ->
     , {platform_etc_dir, "etc"}
     , {platform_lib_dir, "lib"}
     , {platform_log_dir, "log"}
-    , {platform_plugins_dir,  "plugins"}
+    , {platform_plugins_dir,  "etc/plugins"}
```

Comment:
just question, should plugin config stay in plugin dir?

Reply:
to clarify: emqx.conf `plugins.etc_dir` -> app_env `plugins_etc_dir` is used to load plugin configs not this config.
I think it makes sense to have one dir to manage all plugin configs.

as for this config `platform_plugins_dir` which happens to be the same as `plugins.etc_dir` by default.
is currently only used in tests.

as for this specific config: it's mostly not used because